### PR TITLE
Updating Web Overview file

### PR DESCRIPTION
**Pull request by Reni-90**
### Change Description ###

Updated content for the overview.md file.

For images and design reference, see #710

### CODE REVIEW

File: docs/application/web/get-started/overview.md [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
 
-    A mobile application is targeted to run on smart phones, and can take advantage of a wide variety of functionalities available on those devices.
+    Applications targeted to run on smartphones. It can take advantage of the functionalities available on smartphones.
 
-The first application examples show a simple application with a basic UI and minimal functionality. Get familiar with the example first, and
-afterwards you can use it as a basis to build more complex applications.
+The examples in each profile explain a simple application with a basic UI and minimal functionality.
+Get familiar with these examples, and then you can build more complex applications.
 
 ## Richness on your Web applications
```
Comment (Reni-90) [inline docs/application/web/get-started/overview.md:34]: Can we change the heading as:

## Enrich your Web application

Please verify.

File: docs/application/web/get-started/overview.md [hunk 2]
```diff
@@ -1,40 +1,42 @@
 # Get Started with Web Applications
 
-When creating your first Tizen Web application, you can select from the following options, which all use the Web-native languages, such as HTML5, CSS, and JavaScript:
+Tizen Web applications are created using the web-native languages, such as HTML, JavaScript, and CSS that are combined in a package.
```
Comment (journey2w) [inline docs/application/web/get-started/overview.md:3]:
```suggestion
Tizen Web applications are created using not only well-known Web programming languages, such as HTML, CSS, and JavaScript but also Tizen Web API, based on JavaScript, that interacts with the Tizen native subsystems. 
```

If you think the suggestion is a little wordy, you can change it.

File: docs/application/web/get-started/overview.md [hunk 3]
```diff
@@ -1,40 +1,42 @@
...
+
+The Web application uses the Tizen Web framework to interact with the native subsystems. To create your first Tizen Web application, the following profiles are available:
```
Comment (journey2w) [inline docs/application/web/get-started/overview.md:5]:
```suggestion
You can create your first applications as follows:
```

File: docs/application/web/get-started/overview.md (hunk 2 above)
Comment (Reni-90) [reply docs/application/web/get-started/overview.md:3]: Based on your suggestion in https://github.com/Samsung/tizen-docs/pull/1130#discussion_r444121718, and to maintain consistency across all the get started pages, I tried to tweak your suggestion a little:

You can create Tizen Web applications in the web-native languages such as HTML, JavaScript, and CSS. You can also use the Tizen Web APIs that interact with the Tizen native subsystems. To create your first Tizen Web application, you can select from the following profiles:

Please verify and share your valuable feedback.

Comment (journey2w) [reply docs/application/web/get-started/overview.md]: I thought 'web-native languages' expression was awkward so I left suggestion. I know it came from the original one.
In my point of view, there is no problem you rephrase this sentence except for those words.

**Pull request merged**